### DISCUSSION TITLE
プレイリスト・マイリストのような機能が欲しい

### DISCUSSION THREAD

**Discussion by h-yamasawa** [increments/qiita-discussions#411] [要望・アイデア・不具合]
YouTubeでいう公開プレイリストのような機能が欲しいです。
例えば、シリーズのような記事を書いて他人に共有したいときに、そのリストのURLを共有する、のような使い方ができればいいなと思っております。
また、マイページにもそのリストの表示もできると嬉しいです。

今は「ストック」はありますが、自分のストックのURLを他人に共有することはできませんよね…？
（もしできたらすみません）

ご検討の程、よろしくお願いいたします。

**Comment by tanaka-asami415** [comment]
@h-yamasawa 
ご意見ありがとうございます！🙏
今まで考えたことはなかったのですが、今後の開発の際に検討させていただきます！

**Comment by kimkim0814** [comment]
@h-yamasawa
大変お待たせしました！

ご要望いただいた「YouTubeの公開プレイリストのような機能」につきましては、
この度ベータ版としてリリースされた「ストックのシェア機能」で実現可能となりました🎉

これから正式リリースに向けて、改善や機能追加も予定しています！

ぜひ新機能をご活用いただき、気づいたことやご意見をフィードバックいただけると嬉しいです✨
[詳細はこちらをご覧ください。](https://blog.qiita.com/stock-share-beta/)

リクエストいただきありがとうございました！
